aws_s3_rules/aws_s3_unknown_requester_get_object.py

```diff
@@ -5,8 +5,8 @@
         'arn:aws:sts::*:assumed-role/panther-cloud-security-EventProcessorFunctionRole-*/panther-aws-event-processor',
         'arn:aws:sts::*:assumed-role/panther-log-analysis-AthenaApiFunctionRole-*/panther-athena-api',
         'arn:aws:sts::*:assumed-role/panther-log-analysis-RulesEngineFunctionRole-*/panther-rules-engine',
-        'arn:aws:sts::*:assumed-role/panther-snowflake-logprocessing-role-*/snowflake'
-        'arn:aws:sts::*:assumed-role/panther-data-replication-role-*/s3-replication'
+        'arn:aws:sts::*:assumed-role/panther-snowflake-logprocessing-role-*/snowflake',
```

Comment:
Wow, this drove me crazy! Do you think we could have found this with Python linting?

Reply:
Perhaps, although just a quick search found that pylint closed a issue/request for that type of behavior